server/routes/utils/reportHelper.ts

```diff
@@ -103,7 +112,7 @@ export const generatePDF = async (
     //TODO: Add header and footer, phase 2
 
     await browser.close();
-    return { timeCreated, stream, fileName };
+    return { timeCreated, stream: buffer.toString('base64'), fileName };
```

Comment:
1. https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagescreenshotoptions There is an easy way to generate base64 encoded png from Puppeteer. Can we use that for both formats?
2. Since the return type changes, you need to change the function signature accordingly. It will be string(data url) instead of a readable stream.

Reply:
Will change